plugins/sonarqube/dev/index.tsx

```diff
@@ -15,21 +15,61 @@
  */
 
 import { Entity } from '@backstage/catalog-model';
-import {
-  Content,
-  createPlugin,
-  createRouteRef,
-  Header,
-  Page,
-} from '@backstage/core';
-import { createDevApp } from '@backstage/dev-utils';
+import { Content, Header, Page } from '@backstage/core';
+import { createDevApp, EntityGridItem } from '@backstage/dev-utils';
```

Comment:
Btw, the EntityGridItem causes a strange dependency chain: the plugin depends on a core package which itself depends back on a plugin.axbe we should move that component into plugin-catalog-react?

Reply:
It's only ever intended to be used during development and wouldn't want people to use it by mistake, so not sure about that tbh, `dev-utils` is a good home from that PoV. We could also remove it, the main thing it does is avoids a lot of `EntityProvider` wrapping. It also adds a faded label to be able to identify the cards, but I think we can live without that.

Tbh I don't think `catalog-react` needs a dependency on `dev-utils` either tho, since it's not a plugin.

No obvious answer tbh, but perhaps it's safest to just get rid of it in that case 😅

Reply:
Could it be named something with "Test" in the name?

Could it be in `catalog-react` (which is ok to depend on)?